### PR TITLE
[risk=no][no ticket] Minor cleanups of unit tests

### PR DESCRIPTION
**Pull request by jmthibault79**
I cleaned up these test files while integrating the Leo client, but that made a big PR.  Splitting these out to clarify the changes.

No functional changes here.

---
**PR checklist**

- [x] I have included an issue ID or "no ticket" in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [x] I have included a risk tag of the form `[risk=no|low|moderate|severe]` in the PR title as outlined in [CONTRIBUTING.md](https://github.com/all-of-us/workbench/blob/main/.github/CONTRIBUTING.md).
- [ ] I have manually tested this change and my testing process is described above.
- [x] This change includes appropriate automated tests, and I have documented any behavior that cannot be tested with code.
- [ ] I have added explanatory comments where the logic is not obvious.
- One or more of the following is true:
  - [ ] This change is intended to complete a JIRA story, so I have checked that all AC are met for that story.
  - [ ] This change fixes a bug, so I have ensured the steps to reproduce are in the Jira ticket or provided above.
  - [ ] This change impacts deployment safety (e.g. removing/altering APIs which are in use), so I have documented the impacts in the description.
  - [ ] This change includes a new feature flag, so I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later.
  - [ ] This change modifies the UI, so I have taken screenshots or recordings of the new behavior and notified the PO and UX designer in [Slack](https://pmi-engteam.slack.com/archives/C02MWP2RN5P).
  - [ ] This change modifies API behavior, so I have run the relevant E2E tests locally because API changes are not covered by our PR checks.
  - [x] None of the above apply to this change.

### CODE REVIEW

File: api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java [hunk 1]
```diff
@@ -212,37 +211,25 @@ Random random() {
   @Captor private ArgumentCaptor<LeonardoCreateRuntimeRequest> createRuntimeRequestCaptor;
   @Captor private ArgumentCaptor<LeonardoUpdateRuntimeRequest> updateRuntimeRequestCaptor;
 
-  @MockBean LeonardoRuntimeAuditor mockLeonardoRuntimeAuditor;
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java:215]: if we used this field: moved down and grouped with others
if we needed to mock it but didn't need the field: moved into the collective MockBean
else: removed

File: api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java [hunk 2]
```diff
@@ -212,37 +211,25 @@ Random random() {
...
-  @MockBean WorkspaceAuthService mockWorkspaceAuthService;
-  @MockBean LeonardoApiClientFactory mockLeonardoApiClientFactory;
-
   @Qualifier(LeonardoConfig.USER_RUNTIMES_API)
   @MockBean
-  RuntimesApi userRuntimesApi;
-
-  @Qualifier(LeonardoConfig.SERVICE_RUNTIMES_API)
-  @MockBean
-  RuntimesApi serviceRuntimesApi;
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java:229]: unused

File: api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java [hunk 3]
```diff
@@ -212,37 +211,25 @@ Random random() {
...
+  DisksApi mockUserDisksApi;
 
-  @MockBean WorkspaceDao workspaceDao;
-  @MockBean WorkspaceService workspaceService;
+  @MockBean FireCloudService mockFireCloudService;
+  @MockBean WorkspaceAuthService mockWorkspaceAuthService;
+  @MockBean WorkspaceDao mockWorkspaceDao;
+  @MockBean WorkspaceService mockWorkspaceService;
 
-  @Autowired CdrVersionDao cdrVersionDao;
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java:240]: similar treatment of the Autowireds as the MockBeans

File: api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java [hunk 4]
```diff
@@ -336,11 +323,11 @@ public void setUp() throws Exception {
             .setFirecloudName(WORKSPACE_TERRA_NAME)
             .setCdrVersion(cdrVersion)
             .setBillingAccountName(config.billing.initialCreditsBillingAccountName());
-    doReturn(testWorkspace).when(workspaceService).lookupWorkspaceByNamespace(WORKSPACE_NS);
-    doReturn(Optional.of(testWorkspace)).when(workspaceDao).getByNamespace(WORKSPACE_NS);
+    doReturn(testWorkspace).when(mockWorkspaceService).lookupWorkspaceByNamespace(WORKSPACE_NS);
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java:326]: all mockbeans renamed to mockWhatever

File: api/src/test/java/org/pmiops/workbench/leonardo/LeonardoApiClientTest.java [hunk 5]
```diff
@@ -105,27 +103,26 @@ DbUser user() {
...
   @Qualifier(LeonardoConfig.USER_APPS_API)
   @MockBean
-  AppsApi userAppsApi;
+  AppsApi mockUserAppsApi;
 
   @Qualifier(LeonardoConfig.USER_DISKS_API)
   @MockBean
-  DisksApi userDisksApi;
-
-  @MockBean WorkspaceDao workspaceDao;
```
Comment (jmthibault79) [inline api/src/test/java/org/pmiops/workbench/leonardo/LeonardoApiClientTest.java:114]: organized these MockBeans and Autowireds in a similar way

File: api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java [hunk 6]
```diff
@@ -163,31 +159,34 @@ public class RuntimeControllerTest {
 
   @TestConfiguration
   @Import({
-    FakeClockConfiguration.class,
-    RuntimeController.class,
+    AccessTierServiceImpl.class,
```
Comment (evrii) [inline api/src/test/java/org/pmiops/workbench/api/RuntimeControllerTest.java:162]: Alphabetical, I love it!

**Review by evrii — APPROVED** [review]

**Pull request merged**